tests/php/src/Support/SupportDataTest.php

```diff
@@ -151,43 +238,71 @@ public function test_send_data() {
 	}
 
 	/**
-	 * Test get_error_log method.
-	 *
-	 * @covers ::get_error_log
+	 * @covers ::get_data()
+	 * @covers ::get_amp_urls()
 	 */
-	public function test_get_error_log() {
+	public function test_get_data_with_specific_url() {
+
+		$this->create_validated_url();
+
+		$search_urls = home_url( '?s=example' );
+
+		$instance = new SupportData(
+			[
+				'urls' => [
+					$search_urls,
+				],
+			]
+		);
+
+		$output = $instance->get_data();
+
+		$this->assertCount( 1, $output['urls'] );
+		$this->assertEquals(
+			AMP_Validated_URL_Post_Type::normalize_url_for_storage( $search_urls ),
+			$output['urls'][0]['url']
+		);
+	}
+
+	/**
+	 * @covers ::send_data()
+	 */
+	public function test_send_data_with_error() {
+		$callback_wp_remote = static function ( $preempt, $parsed_args ) use ( &$support_data ) {
+
+			$support_data = $parsed_args['body'];
+
+			return [
+				'body' => 'some invalid string',
+			];
+		};
+		add_filter( 'pre_http_request', $callback_wp_remote, 10, 2 );
 
 		$instance = new SupportData( [] );
-		$output   = $instance->get_error_log();
+		$response = $instance->send_data();
 
-		$this->assertArrayHasKey( 'log_errors', $output );
-		$this->assertArrayHasKey( 'contents', $output );
+		$this->assertInstanceOf( 'WP_Error', $response );
```

Comment:
Minor thing: better to use the `class` constant:

```suggestion
		$this->assertInstanceOf( WP_Error::class, $response );
```